src/aleph/vm/utils.py

```diff
@@ -136,3 +139,38 @@ def check_disk_space(bytes_to_use: int) -> bool:
 
 class NotEnoughDiskSpaceError(OSError):
     pass
+
+
+def get_path_size(path: Path) -> int:
+    if path.is_dir():
+        return sum([f.stat().st_size for f in path.glob("**/*")])
+    elif path.is_block_device():
+        return get_block_device_size(str(path))
+    elif path.is_file():
+        return path.stat().st_size
+    else:
+        raise NotImplementedError
```

Comment:
What scenario could lead to this ?

Reply:
Then we should use an `assert False`. This gives the impression that we're going to implement something more later.